Resources/Misc/emsg_list_detailed.txt

```diff
@@ -420,11 +424,11 @@
 	k_EMsgClientFSGetIsFollowingResponse = 7518, // flags: 6, server type: 47
 	k_EMsgClientFSEnumerateFollowingList = 7519, // flags: 6, server type: 47
 	k_EMsgClientFSEnumerateFollowingListResponse = 7520, // flags: 6, server type: 47
-	k_EMsgClientFSOfflineMessageNotification = 7523, // flags: 2, server type: -1
-	k_EMsgClientFSRequestOfflineMessageCount = 7524, // flags: 6, server type: 47
-	k_EMsgClientFSGetFriendMessageHistory = 7525, // flags: 6, server type: 47
-	k_EMsgClientFSGetFriendMessageHistoryResponse = 7526, // flags: 6, server type: 47
-	k_EMsgClientFSGetFriendMessageHistoryForOfflineMessages = 7527, // flags: 6, server type: 47
+	k_EMsgClientChatOfflineMessageNotification = 7523, // flags: 2, server type: -1

```

Comment:
This should have a corresponding EMsg update... we usually leave the old name in as an obsoleted alias for one release.